### PR TITLE
Add more information for README.md

### PR DESCRIPTION
**Pull request by oeular**
Signed-off-by: Xueliang Yang <yangxueliang@cmbchina.com>

### Checklist

I have:

- [ ] Title of the PR starts with type (e.g. `[Addon]` or `[Trait]`).
- [ ] Updated/Added any relevant [documentation] and [examples].
- [ ] Unit/E2E Tests added.

### CODE REVIEW

File: addons/fluxcd/readme.md [hunk 1]
```diff
@@ -16,3 +16,38 @@ Enable fluxcd addon to use these X-definitions
...
+kustomize-controller           kustomize                               ComponentDefinition       kustomize can fetching, building, updating and applying
+                                                                                                 Kustomize manifests from git repo.
+kustomize-controller           kustomize-json-patch                    TraitDefinition           A list of JSON6902 patch to selected target
+kustomize-controller           kustomize-patch                         TraitDefinition           A list of StrategicMerge or JSON6902 patch to selected
+                                                                                                 target
+kustomize-controller           kustomize-strategy-merge                TraitDefinition           A list of strategic merge to kustomize config
+```
+- Source controller
+  - The source-controller is a Kubernetes operator, specialised in artifacts acquisition from external sources such as Git, Helm repositories and S3 buckets. The source-controller implements the source.toolkit.fluxcd.io API and is a core component of the GitOps toolkit.
+  - ![overview](../../addons/fluxcd/source-controller-overview.png)
```
Comment (wonderflow) [inline addons/fluxcd/readme.md:37]: you don't need to copy the image into the addon folder, you can direct use the link from their website.

File: addons/fluxcd/readme.md [hunk 2]
```diff
@@ -16,3 +16,38 @@ Enable fluxcd addon to use these X-definitions
...
+helm-controller                helm                                    ComponentDefinition       helm release is a group of K8s resources from either git
+                                                                                                 repository or helm repo
+kustomize-controller           kustomize                               ComponentDefinition       kustomize can fetching, building, updating and applying
+                                                                                                 Kustomize manifests from git repo.
+kustomize-controller           kustomize-json-patch                    TraitDefinition           A list of JSON6902 patch to selected target
+kustomize-controller           kustomize-patch                         TraitDefinition           A list of StrategicMerge or JSON6902 patch to selected
+                                                                                                 target
+kustomize-controller           kustomize-strategy-merge                TraitDefinition           A list of strategic merge to kustomize config
+```
```
Comment (wonderflow) [inline addons/fluxcd/readme.md:34]: You can use the following syntax to show table in markdown:

```
| Syntax      | Description |
| ----------- | ----------- |
| Header      | Title       |
| Paragraph   | Text        |
```

refer to https://www.markdownguide.org/extended-syntax/#tables

**Review by wonderflow — DISMISSED** [review]

File: addons/fluxcd/readme.md (hunk 2 above)
Comment (oeular) [reply addons/fluxcd/readme.md:34]: OK, Thanks, ^_^

**Review by wangyikewxgm — APPROVED** [review]
Cool job! LGTM

**Review by FogDong — APPROVED** [review]

**Pull request merged**